### ISSUE TITLE
"Green" build on the revision, but marked failed when you click on it?

### ISSUE THREAD

**Issue by joker-eph** [google/llvm-premerge-checks#230]
Every "Harbormaster completed remote builds in B68786: Diff XXXX" mention on this revision are "green": https://reviews.llvm.org/D86120

![Screen Shot 2020-08-19 at 10 23 49 AM](https://user-images.githubusercontent.com/3372300/90669356-61665900-e206-11ea-85b5-6c93dad3b01a.png)

However clicking on it, it shows it as "failed": https://reviews.llvm.org/B68905

![Screen Shot 2020-08-19 at 10 25 45 AM](https://user-images.githubusercontent.com/3372300/90669337-5c090e80-e206-11ea-8966-effb300caae4.png)

**Comment by ChristianKuehnel** [comment]
I guess this is a bug in Phabricator: The build result shown in the
buildable is the same as in buildkite. I have no clue what Phab does
internally...

I've also seen a case the other way round: the log shows a fail but the
buildable a pass.


--
Best,
Christian

Mehdi Amini <notifications@github.com> schrieb am Mi., 19. Aug. 2020, 19:26:

**Comment by metaflow** [comment]
Yes, that seems to be a phabricator/harbormaster behavior of and failures are displayed in a separate section that becomes invisible after submission
![Screenshot from 2020-08-20 12-09-19](https://user-images.githubusercontent.com/254523/90757579-1881d380-e2de-11ea-9837-06cb75a23c94.png) 
so yes, "task failed successfully"

**Comment by joker-eph** [comment]
Unfortunate, I need to look into making this more visible!

**Comment by ChristianKuehnel** [comment]
I just checked the *new* build plan options and they seem to be set reasonably:
https://reviews.llvm.org/harbormaster/plan/5/

No clue what might be causing this...
